src/status_im/transport/db.cljs

```diff
@@ -91,6 +86,10 @@
   (spec/merge :message/message-common
               (spec/keys :req-un [:message.command/content])))
 
+(defmethod content-type "command-request" [_]
```

Comment:
when we receive a legacy content-type "command-request", we validate it the same as content-type "command"